packages/api-gateway/src/drivers/http/adapters/solicitud/diligencia/handlers/create.handlers.diligencia.adapters.js

```diff
@@ -0,0 +1,41 @@
+const { badRequest } = require('@hapi/boom');
+const { Logger, checkers } = require('@siiges-services/shared');
+const errorHandler = require('../../../../utils/errorHandler');
+
+async function createDiligence(fastify, request) {
+  const { ...data } = request.body;
+
+  if (checkers.isFalsy(data?.solicitudId)
+  || checkers.isFalsy(data?.personaId)
+  || checkers.isFalsy(data?.horaInicio)
+  || checkers.isFalsy(data?.horaFin)
+  ) {
+    throw badRequest(`[diligencia/create]: to create a diligencia we need these propeties:
+  solicitudId
+  personaId
+  horaInicio
+  horaFin`);
+  }
+
```

Comment:
I think you could do this in the schema properties to check if something is missing. Check other schemas already done.